watertap/flowsheets/dye_desalination/dye_desalination_global_costing.yaml

```diff
@@ -1,4 +1,4 @@
-base_currency: USD_2020
+base_currency: USD_2023
 base_period: year
 defined_flows:
   electricity:
```

Comment:
Actually, I updated all of these 2020 values to 2023 (just by updating units, not the actual values)

Reply:
And that should probably be done for the analysis results too (unfortunately). The other approach would be to just document what these parameter costs are in 2023 dollars to report _those_ values in our report (rather than report the 2020 values).

Reply:
Oh ok, I see your point. These units should've been in 2023 dollars in the first place, so it's not as simple as convert from USD_2020 to USD_2023. In other words, the values are correct, and only the units need to change